packages/js-sdk/src/sandbox.ts

```diff
@@ -158,8 +252,8 @@ export class Sandbox extends SandboxBase {
    */
   async takeScreenshot(format: 'stream'): Promise<ReadableStream<Uint8Array>>
   async takeScreenshot(format: 'bytes' | 'blob' | 'stream' = 'bytes') {
```

Comment:
I like .screenshot more, reminds me of Playwright

Reply:
screenshot can also be used as a verb afaik.

Reply:
OK